.github/workflows/main.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       ACTIONS_ALLOW_UNSECURE_COMMANDS: true
```

Comment:
Delete these two lines, since they should no longer be needed (and the env setting is unsafe) with the updated setup-miniconda action

Reply:
It would help if I wasn't commiting at 6 am so that camp participants didn't notice that there was wifi.